Tensile/GenerateSummations.py

```diff
@@ -155,44 +179,49 @@ def GenerateSummations(userArgs):
 
         libSolutions = tensileLibrary["solutions"]
 
-        mapper={}
+        libSolutionNames = []
         for s in libSolutions:
-            key=s["name"]
-            value=s["info"]["SolutionNameMin"]
-            mapper[key]=value
+            kenelName=s["name"]
+            libSolutionNames.append(kenelName)
 
-        data_df=pd.read_csv(dataFilePath)
-        working_data = data_df.rename(str.strip,axis='columns').rename(columns=mapper)
+        working_data=pd.read_csv(dataFilePath).rename(str.strip,axis='columns')
 
         index_keys = working_data.SizeL.unique()
         solutionsDF = working_data.filter(like='Cij')
+
         perf_max = solutionsDF.max().max().item()
-    
-        for s in solutionsForSchedule1:
-            s_state = s._state
-            solution_name = s_state["SolutionNameMin"]
-            perf_raw = working_data[solution_name] 
+
+        solutionIndex = 0
+        for s in solutionsForSchedule:
+            s_stateR = solutionStatesR[solutionIndex]
+            kenelName = libSolutionNames[solutionIndex]
+            solutionIndex += 1
+            perf_raw = working_data[kenelName] 
             perf = (1000*index_keys) / perf_raw
             model = np.polyfit(x=index_keys, y=perf, deg=1)
             slope = model[0].item()
             intercept = model[1].item()
             linearModel = {"slope": slope, "intercept": intercept, "max": perf_max}
+            s_stateR["LinearModel"] = deepcopy(linearModel)
+
+        rawLogicData = []
+        rawLogicData.append(deepcopy(versionStringR))
+        rawLogicData.append(deepcopy(scheduleNameR))
+        rawLogicData.append(deepcopy(architectureNameR))
+        rawLogicData.append(deepcopy(deviceNamesR))
+        rawLogicData.append(deepcopy(problemTypeStateR))
+        rawLogicData.append(deepcopy(solutionStatesR))
+        rawLogicData.append(deepcopy(indexOrderR))
+        rawLogicData.append(deepcopy(exactLogicR))
+        rawLogicData.append(deepcopy(rangeLogicR))
+        for idx in range(0, len(otherFieldsR)):
+            rawLogicData.append(otherFieldsR[idx])
```

Comment:
Is the deepcopy needed for the above fields but not `otherFieldsR`?

Reply:
I had a deep copy when reading loading the file but I changed moved it here for consistency.